docs/dev/framework/async-messaging.md

```diff
@@ -0,0 +1,304 @@
+                transports:
+                    - contao_prio_low
+                options:
+                    - --time-limit=60
+                    - --sleep=20
+                autoscale:
+                    desired_size: 20
+                    max: 10
+```
+
+{{% notice idea %}}
+You don't need `Supervisor`, `systemd` or the likes when using the Contao Managed Edition! Just configure a real 
+minutely cron job triggering `contao:cron` and you're good to go!
+{{% /notice %}}
+
+## The priority message interfaces
+
+So we know how the transport works, and we have a solution for running the `messenger:consume` commands. One piece is 
+missing, though: How does Contao know that your message (let's assume a `CreateAsyncZipFileMessage` in this example) 
+should be routed to the `contao_prio_low`, `contao_prio_normal` or `contao_prio_high` transports? The routing part is missing!
+So as an extension developer, you would need to specify the target like so:
+
+```yaml
+framework:
+    messenger:
+        routing:
+            'App\Messenger\CreateAsyncZipFileMessage': contao_prio_high
+```
+
+This would be totally doable using a `Contao Manager Plugin` and adjusting the Symfony Framework configuration, 
+appending your entry. However, because Contao ships with the 3 default priorities, there are also built-in 
+interfaces for those 3 which are then routed automatically:
+
+```yaml
+framework:
+    messenger:
+        routing:
+            'Contao\CoreBundle\Messenger\Message\HighPriorityMessageInterface': contao_prio_high
+            'Contao\CoreBundle\Messenger\Message\NormalPriorityMessageInterface': contao_prio_normal
+            'Contao\CoreBundle\Messenger\Message\LowPriorityMessageInterface': contao_prio_low
+```
+
+{{% notice idea %}}
+Instead of fiddling with the container and the configuration, all you need to do is implement one of the priority
+interfaces and the routing is configured.
+{{% /notice %}}
+
+## Using the entire Contao Managed Edition framework as a developer
+
+The entire setup presented above ensures that - as a developer - you can enjoy a zero-configuration asynchronous 
+message processing setup, provided you have the Contao cronjob framework running. You only need your message and the 
+respective message handler:
+
+1. Register a minutely cronjob for `contao:cron` - aka configure the Contao cron job framework.
+2. Create your message:
+    ```php
+    namespace App\Messenger;
+    
+    use Contao\CoreBundle\Messenger\Message\HighPriorityMessageInterface;
+   
+    class CreateAsyncZipFileMessage implements HighPriorityMessageInterface
+    {
+        public function __construct(public array $fileIds)
+        {
+        }
+    }
+    ```
+3. Create your message handler:
+    ```php
+    namespace App\Messenger;
+        
+    use Symfony\Component\Messenger\Attribute\AsMessageHandler;
+
+    #[AsMessageHandler]
+    class SearchIndexMessageHandler
```

Comment:
```suggestion
    class CreateAsyncZipFileMessageHandler
```

That was likely the intention, I assume? :)

Reply:
Of course🙈